client/my-sites/email/email-management/home/email-list-inactive.jsx

```diff
@@ -1,18 +1,54 @@
 import { Button, Card } from '@automattic/components';
 import { localize } from 'i18n-calypso';
 import React from 'react';
+import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
+import PromoCard from 'calypso/components/promo-section/promo-card';
 import SectionHeader from 'calypso/components/section-header';
 import { canCurrentUserAddEmail } from 'calypso/lib/domains';
-import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+import {
+	emailManagementPurchaseNewEmailAccount,
+	isUnderEmailInbox,
+} from 'calypso/my-sites/email/paths';
 
 class EmailListInactive extends React.Component {
+	getMainHeader() {
+		const { translate } = this.props;
+		const image = {
+			path: emailIllustration,
+			align: 'right',
+		};
+
+		return (
+			<>
+				<PromoCard
+					isPrimary={ true }
+					title={ translate( 'Pick a domain to get started' ) }
+					image={ image }
+				>
+					<p>
+						{ translate(
+							'Pick a domain from your available domains below to add email service to it.'
```

Comment:
I'm not sure if that's normal but the mockup had a slightly different copy:

```suggestion
							'Pick a domain from your available domains below to add an email solution.'
```